apollo-router/src/plugins/telemetry/config_new/connector/events.rs

```diff
@@ -10,6 +10,7 @@ use crate::plugins::telemetry::config_new::extendable::Extendable;
 #[derive(Clone, Deserialize, JsonSchema, Debug, Default)]
 #[serde(deny_unknown_fields, default)]
 pub(crate) struct ConnectorEventsKind {
+    // TODO: I don't think we need this, we should be consistent with what we have today and everything is namespaced like http.*
```

Comment:
Mentioned this in #6067 , but the `StandardEventConfig` and `Condition` types are generic on the `Selector` type, and the `Selector` type has associated types for the request and response types. So in order to support future connectors with request and response types other than `HttpRequest` and `HttpResponse`, there needs to be another nesting level here.

Reply:
Answered [here](https://github.com/apollographql/router/pull/6067#issuecomment-2400027367)